app/src/main/res/values/strings.xml

```diff
@@ -101,26 +102,26 @@
     <string name="confirm_create">Confirm create</string>
     <string name="from">From</string>
     <string name="to">To</string>
-    <string name="pending" translatable="false">Pending</string>
-    <string name="accepted" translatable="false">Accepted</string>
-    <string name="rejected" translatable="false">Rejected</string>
-    <string name="forwarded" translatable="false">Forwarded</string>
+    <string name="pending">Pending</string>
+    <string name="accepted">Accepted</string>
+    <string name="rejected">Rejected</string>
+    <string name="forwarded">Forwarded</string>
     <string name="leave_type">Leave type</string>
     <string name="compensation_time">Compensation time</string>
     <string name="example_time_from">Ex: 2017/05/20 17:30</string>
     <string name="example_time_to">Ex: 2017/05/20 18:30</string>
     <string name="confirm_request_leave">Confirm request leave</string>
     <string name="submit">Submit</string>
     <string name="next">Next</string>
-    <string name="status" translatable="false">Status</string>
-    <string name="all" translatable="false">All</string>
-    <string name="month" translatable="false">Month</string>
-    <string name="date" translatable="false">Date:</string>
-    <string name="date_of_creation" translatable="false">Date of creation:</string>
-    <string name="day_session" translatable="false">Day session</string>
-    <string name="am" translatable="false">am</string>
-    <string name="pm" translatable="false">pm</string>
-    <string name="time_request" translatable="false">Time request:</string>
+    <string name="status">Status</string>
+    <string name="all">All</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.all` appears to be unused
 Unused resources make applications larger and slow down builds.